modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlayerNotificationManager.kt

```diff
@@ -1,22 +1,27 @@
 package au.com.shiftyjelly.pocketcasts.repositories.playback
 
+import android.Manifest
 import android.app.Notification
 import android.content.Context
+import android.content.pm.PackageManager
+import androidx.core.app.ActivityCompat
 import androidx.core.app.NotificationManagerCompat
 
 interface PlayerNotificationManager {
     fun enteredForeground(notification: Notification)
     fun notify(notificationId: Int, notification: Notification)
 }
 
-class PlayerNotificationManagerImpl(context: Context) : PlayerNotificationManager {
+class PlayerNotificationManagerImpl(private val context: Context) : PlayerNotificationManager {
     private val notificationManager = NotificationManagerCompat.from(context)
 
     override fun enteredForeground(notification: Notification) {
         // Not used, just used for a testing point
     }
 
     override fun notify(notificationId: Int, notification: Notification) {
-        notificationManager.notify(notificationId, notification)
+        if (ActivityCompat.checkSelfPermission(context, Manifest.permission.POST_NOTIFICATIONS) == PackageManager.PERMISSION_GRANTED) {
+            notificationManager.notify(notificationId, notification)
+        }
```

Comment:
Should we be ignoring this lint issue, as I believe media sessions notifications are exempt from the permission?

https://developer.android.com/develop/ui/views/notifications/notification-permission#exemptions-media-sessions

Reply:
Good point, thanks for catching deep. It would be an error to use this permission here.